### DISCUSSION TITLE
邮件投递失败dial tcp: lookup smtp.qq.com: i/o timeout

### DISCUSSION THREAD

**Discussion by lujia1009** [Jinnrry/PMail#163] [Q&A]
亲爱的Jinnrry，我在部署Pmail后，发现如下问题：

1，无法接收邮件
2，无法发送邮件
服务器放行了全部端口，并且安装步骤要求的域名解析记录，我也全部解析了。

我使用域名邮箱向QQ发送邮件时，出现了报错：
`[info][2024-07-17 13:09:18][ac110002669751fe699f00019d473bb0][/work/controllers/email/send.go:58]发送邮件`
`[error][2024-07-17 13:09:38][ac110002669751fe699f00019d473bb0][/work/utils/send/send.go:154]2165468532@qq.com%!(EXTRA string=域名mx记录查询失败)`
`[error][2024-07-17 13:09:40][ac110002669751fe699f00019d473bb0][/work/utils/send/send.go:209][0xc000066820] 邮件投递失败dial tcp: lookup smtp.qq.com: i/o timeout`
`[error][2024-07-17 13:09:40][/work/hooks/framework/framework.go:112]params error json: cannot unmarshal object into Go struct field HookDTO.ErrMap of type error`
`[error][2024-07-17 13:09:40][/work/hooks/framework/framework.go:112]params error json: cannot unmarshal object into Go struct field HookDTO.ErrMap of type error`
`[error][2024-07-17 13:09:40][/work/hooks/framework/framework.go:112]params error json: cannot unmarshal object into Go struct field HookDTO.ErrMap of type error`
`smtp/server 2024/07/17 13:14:52 error handling 205.210.31.234:63362: tls: first record does not look like a TLS handshake`

QQ邮箱向域名邮箱发送邮件时，提示“无法与收件人所在的服务商建立连接，connect to smtp.xxx.com: read timeout"
**无法发送邮件，即便向谷歌，微软等邮箱发送，也会出现连接超时**

**我已经在安全组将服务器的出入口端口全部放行**

**Comment by Jinnrry** [comment]
timeout问题一律检查网络，跟程序没有任何关系

**Reply by BitterColaaaaa** [reply]
对网络有啥要求吗？我是香港的机子，也有这个问题
